test/nn/test_edgebank.py

```diff
@@ -0,0 +1,154 @@
+import argparse
+import math
+import os
+import os.path as osp
+import sys
+import timeit
+from pathlib import Path
+
+import pytest
+import torch
+from sklearn.metrics import average_precision_score, roc_auc_score
+from tqdm import tqdm
+
+from torch_geometric.loader import TemporalDataLoader
+from torch_geometric.nn.models import EdgeBankPredictor
+from torch_geometric.testing import withPackage
+
+
+# ==================
+# ==================
+# ==================
+@withPackage('tgb')
+def test_edge_bank_pred():
+    from tgb.linkproppred.dataset import LinkPropPredDataset
+    from tgb.linkproppred.evaluate import Evaluator
+    from tgb.utils.utils import save_results, set_random_seed
```

Comment:
Can we test here instead on a small synthetic dataset instead? I don't think there is a need to test this model on a large real world use-case within unit tests.